src/DistortableCollection.js

```diff
@@ -218,11 +218,35 @@ L.DistortableCollection = L.FeatureGroup.extend({
 
     json.images = json.images.reverse();
     json.avg_cm_per_pixel = this._getAvgCmPerPixel(json.images);
-
+    var jsonImages = json.images;
+    savetoLocalStorage(jsonImages);
     return json;
   },
 });
 
 L.distortableCollection = function(id, options) {
   return new L.DistortableCollection(id, options);
 };
+
+function savetoLocalStorage(jsonImages) {
+  var result = jsonImages.map(img => ({value: img.nodes}));
+  var getImages = localStorage.setItem('locations', JSON.stringify(result));
+  downloadFromLocalStorage(getImages);
+}
+
+function downloadFromLocalStorage(getImages) {
+  var obj = localStorage.getItem('locations');
+  var data = 'text/json;charset=utf-8,' + encodeURIComponent(JSON.stringify(obj, prettyJson));
```

Comment:
From here down is what you want. Let's just connect it to a button! And we can probably skip the prettyJson below - could do it later with a library!